From c1116e0907723916c1d7bc343b9344f51878b87a Mon Sep 17 00:00:00 2001
Message-Id: <c1116e0907723916c1d7bc343b9344f51878b87a.1423872525.git.feitong.yi@intel.com>
In-Reply-To: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
References: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
From: Shashank Sharma <shashank.sharma@intel.com>
Date: Wed, 4 Feb 2015 18:14:29 +0530
Subject: [PATCH 88/95] SQUASHME! [VPG]: adf/intel/hdmi: Suppress the debug
 log for HDMI operations

SQUASHME! - This patch should be squashed into the following existing patch:

	Author: Shashank Sharma <shashank.sharma@intel.com>
	Date:   Thu Jan 29 06:09:00 2015 +0530
	Change-Id: I617af5fe1f3300aa4fb9d4ca0844b4650d3e8f69
	MUST_REBASE [VPG]: adf/intel/hdmi: Enable HDMI support

Issue: GMINL-5478
Change-Id: Ie57d35c00bc48cbdd72e3d229cac2148f0027d4a
Signed-off-by: Shashank Sharma <shashank.sharma@intel.com>
---
 .../adf/intel/core/common/hdmi/gen_hdmi_pipe_ops.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/adf/intel/core/common/hdmi/gen_hdmi_pipe_ops.c b/drivers/video/adf/intel/core/common/hdmi/gen_hdmi_pipe_ops.c
index cba543c..72694dd 100644
--- a/drivers/video/adf/intel/core/common/hdmi/gen_hdmi_pipe_ops.c
+++ b/drivers/video/adf/intel/core/common/hdmi/gen_hdmi_pipe_ops.c
@@ -171,7 +171,7 @@ u32 hdmi_get_vsync_counter(struct intel_pipe *pipe, u32 interval)
 	count += interval;
 	count &= VSYNC_COUNT_MAX_MASK;
 
-	pr_info("ADF: HDMI: vsync count = %#x\n", count);
+	pr_debug("ADF: HDMI: vsync count = %#x\n", count);
 	return count;
 }
 
-- 
1.7.9.5

